tests/utils/mod.rs

```diff
@@ -0,0 +1,13 @@
+use std::path::Path;
```

Comment:
Hmm, I'm personally not really a fan of naming things `utils`, without enough care over time it feels like pieces of code that get hard to categorize turn into _utils_ and it just becomes a dumping ground of unrelated functions, perhaps we can name this something else? eg. `checks`, or `expects`